selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithKeepDirectoryTest.java

```diff
@@ -48,15 +45,18 @@
   @Inject private TestWorkspaceServiceClient workspaceServiceClient;
   @Inject private TestProjectServiceClient projectServiceClient;
   @Inject private DefaultTestUser testUser;
-  @Inject private Dashboard dashboard;
+  @Inject private TestGitHubRepository testRepo;
 
   private TestFactory testFactoryWithKeepDir;
 
   @BeforeClass
   public void setUp() throws Exception {
-    testFactoryWithKeepDir =
-        testFactoryInitializer.fromUrl(
-            "https://github.com/" + gitHubUsername + "/gitPullTest/tree/master/my-lib");
+    // preconditions - create the test repository
```

Comment:
Test repo has actually created earlier, and here we are adding project into the test repo.